src/corehost/cli/deps_resolver.cpp

```diff
@@ -583,9 +615,7 @@ bool deps_resolver_t::resolve_probe_dirs(
             // because of rid-fallback could happen (ex: CentOS falling back to RHEL)
             if ((entry.asset_name == _X("apphost")) && ends_with(entry.library_name, _X(".Microsoft.NETCore.DotNetAppHost"), false))
             {
-                trace::warning(_X("Warning: assembly specified in the dependencies manifest was not found -- package: '%s', version: '%s', path: '%s'"), 
-                    entry.library_name.c_str(), entry.library_version.c_str(), entry.relative_path.c_str());
-                return true;
+                return report_missing_assembly_in_manifest(entry, true);
```

Comment:
Hmm the current code (with the second commit) does have "false" being the default value of that argument, in which case the method will return "false" as well unless the entry is a resource\satellite assembly.